Google.Api.Generator/CodeGenerator.cs

```diff
@@ -142,7 +166,7 @@ internal static class CodeGenerator
                 foreach (var service in fileDesc.Services)
                 {
                     // Generate settings and client code for requested package.
-                    var serviceDetails = new ServiceDetails(catalog, ns, service, grpcServiceConfig);
+                    var serviceDetails = new ServiceDetails(catalog, ns, service, grpcServiceConfig, serviceConfig);
                     allServiceDetails.Add(serviceDetails);
 
                     var ctx = SourceFileContext.CreateFullyAliased(clock, s_wellknownNamespaceAliases);
```

Comment:
Leaving this comment here as I can't write further down.

I think we should keep track of which mixinx we have and add them to the project file we generate (similar to what we do for LRO). Even if we ourselves discard those, we should produce complete project files I believe.

Reply:
Ooh yes, good plan. Will do that next week.

Reply:
This is now done.